pycqed/measurement/waveform_control/pulsar.py

```diff
@@ -657,9 +686,62 @@ def g():
                             .format(int(id[2])-1))/2
             else:
                 return lambda: 1
+        elif par == 'amplitude_scaling' and id[-1] != 'm':
+            awg = int((int(id[2:]) - 1) / 2)
+            output = (int(id[2:]) - 1) - 2 * awg
+            def g():
+                return obj.get(f'awgs_{awg}_outputs_{output}_amplitude')
         else:
             raise NotImplementedError('Unknown parameter {}'.format(par))
-        return g 
+        return g
+
+    @staticmethod
+    def _hdawg_mod_setter(obj, awg_nr):
```

Comment:
Again can't judge the details of this function because i'm not familiar with the HDAWG parameters but I trust it's correct. 

Is there documentation for these parameters somewhere? Maybe you can a docstring for this method with a link to that if it exists. If not, this is probably not the place to add it but I personally have no idea where I can learn about the available parameters of ZI devices.